### PR TITLE
udpdate readme

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -17,7 +18,31 @@ Tech this uses [thnx]:
...
+```
+
+Locally exclude built files and stuff. Edit `.git/info/exclude`:
+
+```
+composer.lock
+vendor
+dist/
+```
+
```
Comment (mixcas) [inline README.markdown]: This is really good!

Comment (varffs) [inline README.markdown]: From https://help.github.com/articles/ignoring-files/ it seems like unless we had build scripts at least one of us would need to be committing these files for remotes to work, so not sure this is useful right now

**Comment by mixcas** [comment]
https://help.github.com/articles/ignoring-files/

File: README.markdown (hunk 1 above)
Comment (mixcas) [inline README.markdown]: Thats only for developing the `globie-wordpress-starter-pack` not for making new themes

Comment (varffs) [inline README.markdown]: ah yes thats good

**Pull request merged**